docs/make-docs

```diff
@@ -6,6 +6,15 @@
 # [Semantic versioning](https://semver.org/) is used to help the reader identify the significance of changes.
 # Changes are relevant to this script and the support docs.mk GNU Make interface.
 #
+# ## 8.1.0 (2024-08-22)
+#
+# ### Added
+#
+# - Additional website mounts for projects that use the website repository.
+#
+#   Mounts are required for `make docs` to work in the website repository or with the website project.
+#   The Makefile is also mounted for convenient development of the procedure that repository.
```

Comment:
I think this sentence is missing a word but I can't quite figure out which one.

Reply:
Oh yeah, probably:

```suggestion
#   The Makefile is also mounted for convenient development of the procedure in that repository.
```

Reply:
If we make that change, the automated workflow will overwrite it though. I can fix that upstream when I next make a meaningful change to the script. Thanks for checking it :)